src/modules/areas.js

```diff
@@ -127,12 +128,15 @@ export function getAreas() {
 }
 
 // POST name, geostore ID
-export function saveArea(area) {
-  const url = `${API_BASE_URL}/area`;
-  const body = new FormData();
-  body.append('name', area.name);
-  body.append('geostore', area.geostore);
-  return (dispatch, state) => {
+export function saveArea(area, node) {
+  return async (dispatch, state) => {
+    const url = `${API_BASE_URL}/area`;
+    const body = new FormData();
+    const blob = await domtoimage.toBlob(node);
```

Comment:
This wont work as it grabs the height and width as an overlay to the node -> for example the top left square of 300px x 300px.